From 51fb6da1551a14197e6d6d74da41bbc3c89e909a Mon Sep 17 00:00:00 2001
From: Florin Sarbu <florin@balena.io>
Date: Thu, 13 Jun 2019 13:21:45 +0200
Subject: [PATCH] rtk_btusb: Adapt to kernel 4.4.180

Upstream-Status: Pending
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 drivers/bluetooth/rtk_btusb.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/bluetooth/rtk_btusb.c b/drivers/bluetooth/rtk_btusb.c
index 5db1884..f99a788 100644
--- a/drivers/bluetooth/rtk_btusb.c
+++ b/drivers/bluetooth/rtk_btusb.c
@@ -3065,7 +3065,7 @@ static int btusb_open(struct hci_dev *hdev)
     int err = 0;
 
     RTKBT_INFO("%s: Start, PM usage count %d", __func__,
-            atomic_read(&(data->intf->pm_usage_cnt)));
+            atomic_read(&(data->intf->dev.power.usage_count)));
 
     err = usb_autopm_get_interface(data->intf);
     if (err < 0)
@@ -3096,7 +3096,7 @@ static int btusb_open(struct hci_dev *hdev)
 done:
     usb_autopm_put_interface(data->intf);
     RTKBT_INFO("%s: End, PM usage count %d", __func__,
-            atomic_read(&(data->intf->pm_usage_cnt)));
+            atomic_read(&(data->intf->dev.power.usage_count)));
     return 0;
 
 failed:
@@ -3104,7 +3104,7 @@ failed:
     clear_bit(HCI_RUNNING, &hdev->flags);
     usb_autopm_put_interface(data->intf);
     RTKBT_ERR("%s: Failed, PM usage count %d", __func__,
-            atomic_read(&(data->intf->pm_usage_cnt)));
+            atomic_read(&(data->intf->dev.power.usage_count)));
     return err;
 }
 
@@ -3611,7 +3611,7 @@ static void btusb_waker(struct work_struct *work)
     int err;
 
     RTKBT_DBG("%s: PM usage count %d", __func__,
-            atomic_read(&data->intf->pm_usage_cnt));
+            atomic_read(&data->intf->dev.power.usage_count));
 
     err = usb_autopm_get_interface(data->intf);
     if (err < 0)
-- 
2.7.4

